Quantum-Control-Applications/Superconducting/Single Fixed Transmon/allxy.py

```diff
@@ -341,32 +128,6 @@
 
     plt.figure()
     plt.plot(I)
-
-    plt.figure()
```

Comment:
Oh really? WHy don't we directly plot the amplitude and phase then?

Reply:
Because just like Rabi, it's not necessarily the amplitude that has the information. Sometimes it's easier to look at the separate I & Q values.

Reply:
Ok, I thought the amplitude and phase together have the full information, but I split the plot in I and Q separately then

Reply:
Well you are right, the question is what is the most convenient thing to look at.
But in any case, if we plot amplitude and phase, we do it in two figures because they have different scales.
Same thing with I & Q :)

Reply:
Gotcha :)